data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,495 @@ mission "Unfettered returning home"
+				accept
+	on enter
+		system
+			attributes "unfettered"
+		conversation
+			branch "wrong flagship"
+				not "flagship model: Anomalocaris"
+			branch "not single ship"
+				"total ships" != 1
+			action
+				"old reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)"
+				"reputation: Hai (Unfettered)" >?= 0
+			`As you enter the system, you prepare for a fight to get the cargo through, but it is surprisingly quiet. For the first time, entering Unfettered space does not mean getting shot at by missiles from all sides. They're not hailing you either. Perhaps they think you're one of the Hai aiding them, especially given your cargo and the ship you're flying.`
+				accept
+			label "not single ship"
+			`You enter the system, and, for a brief moment, nobody is shooting you.`
+			`	The Unfettered quickly change their mind, however, as soon as your escorts enter the system. They happily head your way, weapons ready to loot your cargo. It becomes clear you should have come with only the Anomalocaris, as the associates suggested.`
+				decline
+			label "wrong flagship"
+			`You enter the system, and the Unfettered open fire as usual. Perhaps they would have acted differently if you had used the Anomalocaris as a flagship, but it is too late to find out now.`
+				decline
+	on fail
+		"reputation: Hai (Unfettered)" <?= -1000
+		dialog `The Unfettered and their associates are clearly not happy when they somehow find out you didn't deliver the cargo in time. The message they send you is short: "You will have to prove yourself another way."`
+	on visit
+		dialog `You have reached <planet>, but you either did not bring the Anomalocaris, some of the cargo was on another ship, or you are not using the Anomalocaris as your flagship. You should park your fleet and only use the Anomalocaris.`
+	to complete
+		has "flagship model: Anomalocaris"
+	on complete
+		dialog `You receive a message from the associates you bought the ship from: "Thank you for helping our brethren, <last>. We will not forget this. Please accept these <payment>."`
+		payment 1500000
+		log "Delivered a pressing food shipment to the Unfettered Hai using an Anomalocaris. They're no longer shooting at me... for now, at least."
+
+
+
+# No category, meaning you can't remove it from the Anomalocaris
+outfit "Unfettered Landing Permit"
+	"maintenance costs" 650
+	description "This item is needed in order to land on Unfettered planets, and its loss will cancel the arrangement."
+
+mission "Hai Legacy 3"
+	landing
+	name "Ancient claim"
+	description "Beat the Lightning Bugs in combat. You only have one day, and becoming disabled will result in losing the fight."
+	source "Firelode"
+	deadline 2
+	to offer
+		has "flagship model: Anomalocaris"
```

Comment:
This seems redundant, since the previous mission requires the flagship to be the Anomalocaris to complete, and this mission offers right after that while you're still on the planet.

Reply:
```suggestion
```
yeah it is you're right, this is already triggering on landing too